bundles/org.openhab.binding.opengarage/README.md

```diff
@@ -23,13 +23,13 @@ As a minimum, the IP address is needed:
 
 ## Channels
 
-| channel              | type          | description                                            |
-|----------------------|---------------|--------------------------------------------------------|
-| distance             | Number:Length | Distance reading from the OpenGarage controller (default in cm)       |
-| status-switch        | Switch        | Door status (OFF = Closed, ON = Open)                  |
-| status-contact       | Contact       | Door status (Open or Closed)                           |
-| status-rollershutter | Rollershutter | Door status (DOWN = Closed, UP = Open)                 |
-| vehicle              | String        | Report vehicle presence from the OpenGarage controller |
+| channel              | type          | description                                                                           |
+|----------------------|---------------|---------------------------------------------------------------------------------------|
+| distance             | Number:Length | Distance reading from the OpenGarage controller (default in cm)                       |
+| status-switch        | Switch        | Door status (OFF = Closed, ON = Open), set "invert=true" on channel to invert switch  |
+| status-contact       | Contact       | Door status (Open or Closed)                                                          |
+| status-rollershutter | Rollershutter | Door status (DOWN = Closed, UP = Open)                                                |
+| vehicle-status       | Number        | Report vehicle presence (0=Not Detected, 1=Detected, 2=Unknown)                       |
```

Comment:
Why is the channel name changed? Was this a bug in the readme?

Reply:
I changed that in another PR that was merged , but I must of missed committing the README change in it ( i probably did a `commit -s ./src` )  .   The string channel is depreciated, although still there for backwards compatibility, in favor of this number type which has state options in the channel xml definition.